Vector.java

```diff
@@ -0,0 +1,41 @@
+import java.io.*;
+import java.util.*;
+import java.lang.Math; 
+
+public class Vector {
+	
+	List<Double> numbers;
+		
+	Vector (List<Double> numbers) {
+			
+		numbers = numbers;
+			
+	}
+		
+	public double distance(Vector vector2) { // returns distance between two vectors
+		
+		double squared_distance = 0;
```

Comment:
need error checks here. What if the other vector is not of the same length as this one?  If that's the case, throw an Exception